docs/generate-time-series/go.rst

```diff
@@ -380,27 +408,30 @@ will open up a map view showing the current position of the ISS:
 
 .. image:: ../_assets/img/generate-time-series/map.png
 
-You can find the full version of the script `here`_. 
+.. TIP::
+
+    The ISS passes over large bodies of water. If the map looks empty, try
+    zooming out.
 
 
 .. _compilation: https://www.geeksforgeeks.org/difference-between-compiled-and-interpreted-language/
 .. _context: https://golang.org/pkg/context/
 .. _CrateDB Admin UI: https://crate.io/docs/clients/admin-ui/en/latest/
 .. _create a table: https://crate.io/docs/crate/reference/en/latest/general/ddl/create-table.html
-.. _INSERT: https://crate.io/docs/crate/reference/en/latest/general/dml.html#inserting-data
```

Comment:
@infoverload re-ordered the list of URLs. our current style guide indicates that these lists should be ordered alphabetically (case-insensitive)